tensorflow/compiler/xla/service/gpu/amdgpu_compiler.cc

```diff
@@ -92,6 +95,10 @@ Status AMDGPUCompiler::OptimizeHloPostLayoutAssignment(
       /*allow_mixed_precision=*/false,
       LayoutAssignment::InstructionCanChangeLayout);
 
+  pipeline.AddPass<ReductionDegenerateDimRemover>();
+  pipeline.AddPass<ReductionLayoutNormalizer>();
+  pipeline.AddPass<ReductionDimensionGrouper>();
```

Comment:
I guess sharing this registration between nvptx_compiler and amdgpu_compiler could be done in a separate PR?

Reply:
Will rebase and rename this PR to test only and file another PR on the three passes.

Reply:
#36191 filed